src/pages/stop/stop.component.ts

```diff
@@ -163,7 +178,12 @@ export class StopComponent {
   toggleStopHeart(): void {
     // console.log('toggling', stop.Description);
     this.favoriteStopSvc.toggleFavorite(this.stopId, this.stop.Description);
+    if (this.liked) {
+      this.showToast('Stop added to Favorites');
+    }
+    else this.showToast('Stop removed from Favorites');
```

Comment:
Sorry. Yeah, I totally see it. I'm just not used to writing in typescript/javascript. I'll look it up.